Library/Homebrew/rubocops/cask/on_system_conditionals.rb

```diff
@@ -39,13 +39,43 @@ def on_cask(cask_block)
 
             audit_on_system_blocks(stanza.stanza_node, stanza.stanza_name)
           end
+
+          audit_arch_conditionals(cask_body)
```

Comment:
For formulae, we only require using `on_{system}` blocks over `if` statements in Homebrew/core. Should we do the same here and not require that people switch?

Reply:
I think it's worth making people switch in taps too for consistency (and we have some autocorrection, right)?